.hlint.yaml

```diff
@@ -6,7 +6,7 @@
 # Settings
```

Comment:
Since things have changed in `main` since the last rebase, I think we could use interpolation here as well:

* `Buildable` instance for `InternalCommandsError` and `CofferError`
* `Buildable` instance for `VaultError`
* `CLI.Parser.readSum`, `readFieldInfo`, `readSort`, `readFilter`, `readQualifiedPath`, `configPathParser`

Reply:
Done